README.md

```diff
@@ -31,16 +31,19 @@ It should be possible to build everything without Qt Creator, but the guide assu
    Client executables will appear in the `exec` folder, and the server executable will appear in the `griefly-server` folder.  
    Client executables depend from various dlls, so it is needed to manually place them to the `exec` folder.
 
-**Note:** It is supposed to perform build from an active git repository (`git describe --tags --abbrev=4` will be called during the process).
+**Note:** It's supposed to be built from an active git repository (`git describe --tags --abbrev=4` will be called during the process).
```

Comment:
I am not sure why you have changed "It is" to "It's". In other places in the `README.md` reductions are not used.
Maybe there are some reason to use it here?